docs/photoniq/fingerprinting/fingerprinting-best-practices.md

```diff
@@ -3,16 +3,17 @@ sidebar_position: 80
 title: Fingerprint Best Practices
 ---
 
-This page describes best practices to adopt in order to get the most out of Dynamic Fingerprinting.
+
+Follow these best practices to maximize the benefits offered by the dynamic fingerprinting service.
 
 ### First-Party vs. Third-Party Cookies
 
-The Digital Fingerprint service uses [third-party cookies](first-third-party-cookie) by default, which may affect the accuracy of the identification process. To ensure high levels of identification accuracy, it is recommended to use a customer subdomain or a cloud proxy (such as Akamai). By employing this approach, the cookies set by the Digital Fingerprint service will be treated as [first-party cookies](first-third-party-cookie), which will enhance the identification precision.
+The Digital Fingerprint service uses [third-party cookies](first-third-party-cookie) by default, which may affect the accuracy of the identification process. To ensure high levels of identification accuracy, we recommend using a customer subdomain or a cloud proxy (such as Akamai). This approach treats the Fingerprint service cookies as [first-party cookies](first-third-party-cookie), enhancing identification precision.
 
 ### Fingerprint Browser Agent on Page Load
 
-The JavaScript agent is loaded at page load time, and the function to get the visitor ID is run right away. For more information, refer to [Fingerprint on Page Load](get-started-fingerprinting.md#fingerprint-on-page-load).
+The script used to activate the agent affects the performance of the Fingerprint service. For example, activating the Javascript agents at page load time also runs a function that generates the `visitorId`, improving the speed and performance, unlike the script that activates the agent after clicking on a button, which takes longer. 
```

Comment:
I dont think we should say it like this in customer facing docs.
The point is that with FPS, the end user is free to choose when they want to call the service - either on page load or button click, etc. It is the way it is for flexibility and has no direct correlation with FPS performance.